web/nanoc.yaml

```diff
@@ -7,6 +7,8 @@ prune:
 data_sources:
   - type: filesystem
     encoding: utf-8
+  - type: releases
+    items_root: /release
```

Comment:
```suggestion
    items_root: /releases
```
Typo?

Reply:
Looking at https://docs.theforeman.org/release/3.10/ I think it should be `/release`.